src/ui/components/SensorChartRow.tsx

```diff
@@ -41,11 +27,9 @@ export const SensorChartRow: FC<SensorChartRowProps> = React.memo(({ id, onPress
   const name = useSelector((state: RootState) => SensorSelector.getName(state, { id }));
   const hasData = useSelector((state: RootState) => SensorStatusSelector.hasData(state, { id }));
 
-  useOnMount([fetchStatus, fetchChartData]);
-
   return (
     <SensorRowLayout
-      onPress={hasData && !isLoadingChartData ? () => onPress() : undefined}
+      {...(hasData && !isLoadingChartData ? { onPress } : {})}
```

Comment:
ts thing which I was getting errors from if `onPress` was used as a prop when it was undefined